cfgov/v1/atomic_elements/molecules.py

```diff
@@ -12,10 +12,14 @@ def isRequired(field_name):
 
 
 class HalfWidthLinkBlob(blocks.StructBlock):
+    h_text = ('A list of icon names can be obtained at: '
```

Comment:
A bit late to this, but I'd agree that if this is going to be pulled out as a separate variable it should be given a more precise name, e.g. `sub_heading_icon_help_text`. That being said, is the reason to pull this out solely for formatting purposes? Doing it this way means there's now a `h_text` class variable on the class itself which doesn't really serve much purpose.

A PEP8-compliant way of doing this might be something like:

```py
    sub_heading_icon = blocks.CharBlock(
        required=False,
        label='Sub heading icon',
        help_text=(
             'A list of icon names can be obtained at: '
             'https://cfpb.github.io/capital-framework/components/cf-icons/. '
             'Example: linkedin-square, facebook-square, etc.'
        )
    )
```